src/pages/workspaces/workspace/analysis/modals/persistent-disk-controls.test.ts

```diff
@@ -155,5 +154,17 @@ describe('compute-modal-component', () => {
       // Assert
       expect(updateComputeConfig).toBeCalledWith({ displayName: 'Balanced', label: 'pd-balanced', regionToPricesName: 'monthlyBalancedDiskPrice' })
     })
+
+    it('should call updateComputeConfig with proper value on changing size', async () => {
```

Comment:
Could we add a little more info here? It's confusing that it's being called with 500 when the user types 0.

It makes sense that it starts at 50 and then 1 0 would make it 500. Just wondering if we can clarify that.

Reply:
ohhh yeah I was missing the `userEvent.clear()` method for this one, updated